stdlib/platform.pyi

```diff
@@ -37,9 +37,7 @@ def java_ver(
     release: str = "", vendor: str = "", vminfo: tuple[str, str, str] = ..., osinfo: tuple[str, str, str] = ...
 ) -> tuple[str, str, tuple[str, str, str], tuple[str, str, str]]: ...
 def system_alias(system: str, release: str, version: str) -> tuple[str, str, str]: ...
-def architecture(
```

Comment:
part of me wants to keep this as an easter egg

Reply:
It's iconic